Add Appointment Scheduling usecase

isuruuy429 opened the pull request:
## Purpose
This PR contains the following. 
- Front-end implementation of the appointment-scheduling 
- Ballerina medconnect service
- README.md files

isuruuy429 committed:
Move urlConfigs.ts file inside the src folder

isuruuy429 committed:
Update the README file

sameeragunarathne commented on use-cases/schedule-appointment/medconnect-service/appointment_api_config.bal:
```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```
Licence is Apache 2.0. Let's add the correct licence to all the source files

isuruuy429 replied:
Fixed in https://github.com/wso2/solutions-healthcare-demos/pull/8/commits/1f5aaedc782d76068b76a5b7936c80a5eece4298

sameeragunarathne commented:
Did you check copyrights for these images? or generated?

isuruuy429 committed:
Update the Appointment booking image

isuruuy429 commented:
> Did you check copyrights for these images? or generated?

One image was generated. I removed the other image and added a generated image. 

https://github.com/wso2/solutions-healthcare-demos/pull/8/commits/72c5d7497da1649e14f3097611e53d55a69b694b

isuruuy429 committed:
Update license header
